src/diffusers/models/controlnet_flax.py

```diff
@@ -154,7 +156,8 @@ class FlaxControlNetModel(nn.Module, FlaxModelMixin, ConfigMixin):
     only_cross_attention: Union[bool, Tuple[bool]] = False
     block_out_channels: Tuple[int] = (320, 640, 1280, 1280)
     layers_per_block: int = 2
-    attention_head_dim: Union[int, Tuple[int]] = 8
+    attention_head_dim: Union[int, Tuple[int]] = (8,)
```

Comment:
Wouldn't this break when there are more than 1 down blocks?

Reply:
Yes 100%, always forgetting that Flax is a data class

Reply:
Also, we should have probably added tests for ControlNetFlax so that this would give a red CI